lib/windowsbasetest.pm

```diff
@@ -47,32 +47,31 @@ sub select_windows_in_grub2 {
 sub open_powershell_as_admin {
     my ($self, %args) = @_;
 
-    send_key_until_needlematch "windows-quick-features-menu", 'super-x';
-    wait_still_screen stilltime => 2, timeout => 15;
     #If using windows server, and logged with Administrator, only open powershell
     if (get_var('QAM_WINDOWS_SERVER')) {
-        send_key 'shift-a';
+        send_key_until_needlematch 'windows-quick-features-menu', 'super-x';
+        wait_screen_change { send_key('shift-a') };
         wait_screen_change { assert_and_click('window-max') };
-        assert_screen 'windows_server_powershel_opened', 30;
+        assert_screen 'windows_server_powershell_opened', 30;
     } else {
-        send_key 'shift-a';
+        if (check_var('WIN_VERSION', '10')) {
+            send_key_until_needlematch 'windows-quick-features-menu', 'super-x';
+            wait_screen_change { send_key('shift-a') };
+        } else {
```

Comment:
Maybe, instead of `else`, to make clear for which condition the code is executed, we could do `if (check_var('WIN_VERSION', '11')) {` and then, if you want, another `else` with version not supported.. 
But feel free to ignore.

Reply:
Good idea. I've added an `elsif` and a `die` in case none of the versions are correct.